package.json

```diff
@@ -21,7 +21,7 @@
     "test-base": "npm run test-js-base && npm run test-python-base && npm run test-php-base",
     "test-js-base": "mocha js/test/base/test.base.js --reporter ololog/reporter",
     "test-python-base": "python python/test/test_decimal_to_precision.py",
-    "test-php-base": "php -f php/test/decimal_to_precision.php",
+    "test-php-base": "php -f php/test/decimal_to_precision.php && php -f php/test/test_crypto.php",
```

Comment:
Just the PHP? ;)

Reply:
i also added the js to `test-base.js`, the python `sha3` (keccak) still relies on web3, for this reason it cannot be included in the `npm run test`.

Reply:
also i need the `decimal_to_bytes` from this branch since everything is little endian encoded with a varying amount bytes per thing (